pytorch_lightning/utilities/memory.py

```diff
@@ -135,8 +135,11 @@ def get_gpu_memory_map() -> Dict[str, float]:
         A dictionary in which the keys are device ids as integers and
         values are memory usage as integers in MB.
     """
+    nvidia_smi_path = shutil.which("nvidia-smi")
+    if nvidia_smi_path is None:
+        raise FileNotFoundError("nvidia-smi: command not found")
```

Comment:
could you also update the docstring for the function that this raises a FileNotFoundError?